templates/superwerker.template.yaml

```diff
@@ -31,42 +31,41 @@ Metadata:
     ParameterLabels:
       Domain:
         default: Domain for automated DNS configuration
-      Subdomain:
-        default: Subdomain for automated DNS configuration
-      QSS3BucketName:
-        default: Quick Start S3 Bucket Name
-      QSS3BucketRegion:
-        default: Quick Start S3 Bucket Region
-      QSS3KeyPrefix:
-        default: Quick Start S3 Key Prefix
       IncludeBudget:
-        default: Enable Budget Alarm for monthly AWS spendings
+        default: Include AWS Budgets alarm
       IncludeControlTower:
-        default: Enable AWS Control Tower
+        default: Include AWS Control Tower
       IncludeGuardDuty:
-        default: Enable Amazon GuardDuty
+        default: Include Amazon GuardDuty
       IncludeSecurityHub:
-        default: Enable AWS Security Hub
+        default: Include AWS Security Hub
       IncludeBackup:
-        default: Enable Automated Backups
+        default: Include Automated Backups
       IncludeServiceControlPolicies:
-        default: Enable Service Control Policies in AWS Organizations
-
+        default: Include service control policies
+      QSS3BucketName:
+        default: Quick Start S3 bucket name
+      QSS3BucketRegion:
+        default: Quick Start S3 bucket Region
+      QSS3KeyPrefix:
+        default: Quick Start S3 key prefix
+      Subdomain:
+        default: Subdomain for automated DNS configuration
 Parameters:
   Domain:
     Type: String
-    Description: Domain used for RootMail feature
+    Description: Domain used for root mail feature
```

Comment:
`RootMail` is the product name. I would not change this.

Reply:
Yes, TBH I did not look at the changes in detail. My primary goal was to gain parity again. Would this be a blocker for merging this PR? We could also open a PR at the QuickStarts repo to name it back to `RootMail` over there so we decouple the two concerns.